purchases/src/main/kotlin/com/revenuecat/purchases/coroutinesExtensions.kt

```diff
@@ -0,0 +1,20 @@
+package com.revenuecat.purchases
+
+import kotlin.coroutines.resume
+import kotlin.coroutines.resumeWithException
+import kotlin.coroutines.suspendCoroutine
+
+/**
+ * Get latest available customer info.
+ * Coroutine friendly version of [Purchases.getCustomerInfo].
+ *
+ * @return The [CustomerInfo] or a [PurchasesException] with the [PurchasesError]
+ */
+suspend fun Purchases.awaitCustomerInfo(): CustomerInfo {
```

Comment:
We should add this to the API tests. Also check if Java adds anything... I think it shouldn't but better check to make sure.

Reply:
Good point, I will add those

Reply:
Added some API tests in [`e3a86c7` (#1012)](https://github.com/RevenueCat/purchases-android/pull/1012/commits/e3a86c791fecb49d7536d7673dae4ab2747c8b56) . Do you think we should add those Java API tests? Tbh, not sure if anyone is really going to be using the coroutines from Java

Reply:
Hmm I think it's probably fine to not include those. I agree that it would be weird for Java users to use this API.

Reply:
I totally agree. I'd expect very few (none?) clients working with Coroutines from Java 👍